src/main/java/com/google/summit/translation/Translate.kt

```diff
@@ -227,7 +227,7 @@ class Translate(val file: String, private val tokens: TokenStream) : ApexParserB
   /** Translates the 'typeDeclaration' grammar rule and returns an AST [TypeDeclaration]. */
   override fun visitTypeDeclaration(ctx: ApexParser.TypeDeclarationContext): TypeDeclaration {
     // Check mutual exclusivity in grammar
```

Comment:
These comments are used inconsistently - only 5 instances. Should they be removed?

Reply:
I think you made the code so much clearer that they aren't adding anything anymore. Up to you whether you want to remove.